src/components/basic-table/components/cell/StyledCell.tsx

```diff
@@ -0,0 +1,14 @@
+import { themedStyled } from '../../../../theme';
+import { TABLE_ROW_HEIGHT } from 'models';
```

Comment:
```suggestion
import { TABLE_ROW_HEIGHT } from '../../../models';
```
This should be a relative import if not it will break in next. 
I will create a pr with some lintin rule so this error will also popup inside tacugama